source/gui/settingsDialogs.py

```diff
@@ -3032,6 +3032,18 @@ def __init__(self, parent):
 		)
 		self.wasapiCheckBox.defaultValue = self._getDefaultValue(
 			["audio", "wasapi"])
+		# Translators: This is the label for a checkbox control in the
+		#  Advanced settings panel.
+		label = _("Volume of NVDA sounds follows voice volume (requires WASAPI)")
+		self.soundVolFollowCheckBox: wx.CheckBox = audioGroup.addItem(
+			wx.CheckBox(audioBox, label=label)
+		)
+		self.bindHelpEvent("SoundVolumeFollowsVoice", self.soundVolFollowCheckBox)
+		self.soundVolFollowCheckBox.SetValue(
+			config.conf["audio"]["soundVolumeFollowsVoice"]
+		)
+		self.soundVolFollowCheckBox.defaultValue = self._getDefaultValue(

```

Comment:
we should probably disable/enable `soundVolFollowCheckBox` when `wasapiCheckBox` is toggled. 

I think this pattern exists in the Braille settings panel (e.g. `onShowCursorChange`)